pulp_container/app/tasks/sync_stages.py

```diff
@@ -175,11 +169,25 @@ async def run(self):
                 saved_artifact, content_data, raw_data, response = await artifact
 
                 digest = response.artifact_attributes["sha256"]
+                tag_name = response.url.split("/")[-1]
+
+                # Look for cosign signatures
+                # cosign signature has a tag convention 'sha256-1234.sig'
+                if self.signed_only and not signature_source:
+                    if not tag_name.enswith(".sig") and digest + ".sig" not in tag_list:
```

Comment:
Can we verify if the tag begins with "sha256" too?

Reply:
by conventions tag cannot begin with sha256

Reply:
nevermind, i misread your ask in this specific context.
Tag can have a format of 'sha256-1234'  but not 'sha256:1234'.